src/main/java/edu/nextstep/racing/domain/Winner.java

```diff
@@ -0,0 +1,51 @@
+package edu.nextstep.racing.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * author       : gwonbyeong-yun <sksggg123>
+ * ------------------------------------------
+ * | email        : sksggg123               |
+ * | github       : github.com/sksggg123    |
+ * | blog         : sksggg123.github.io     |
+ * ------------------------------------------
+ * project      : java-racingcar
+ * create date  : 2019-06-22 19:52
+ */
+public class Winner {
+
+    private final int winScore;
+    private Cars winner;
+
+    public Winner(Cars winner, int winScore) {
+        this.winner = winner;
+        this.winScore = winScore;
+    }
+
+    public static Winner checkPosition(List<Car> candinateWinner) {
```

Comment:
전반적으로 중복된 코드가 많아요. 일급 컬렉션인 `Cars`를 사용해 보세요.

```suggestion
    public static Winner checkPosition(Cars candinateWinner) {
```

Reply:
일급컬렉션 Cars를 활용하도록 수정하였습니다.